CI_FIXES_SUMMARY.md

```
@@ -0,0 +1,109 @@
+- `scripts/lint.sh`
+
+**Verification**:
+```bash
+CI=true bash scripts/lint.sh
+# Output: All checks passed!
+```
+
+### 2. Async Startup Table Creation ✅ FIXED
+
+**Problem**: TypeScript Playwright tests failed because backend startup crashed with:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)